### DISCUSSION TITLE
Question from dumb v3 user

### DISCUSSION THREAD

**Discussion by ClaudioFloreani** [mikefarah/yq#784] [Q&A] [answered]
I was using the depracted v3 command
`yq read --stripComments config/file.yml Entity`
in several scripts.

After half an hour of searching through the docs at https://mikefarah.gitbook.io/yq/v/v4.x, all I was able to do is
`yq eval '.Entity' config/file.yml` but I didn't find out how to put "stripComments" into the expression language.

Any advice?

Thanks

**Comment by mikefarah** [comment, marked as answer]
this should do the trick:

```
yq eval '.Entity | ... comments=""' config/file.yml
```

For the record, documented here: https://mikefarah.gitbook.io/yq/v/v4.x/operators/comment-operators#remove-all-comments

**Reply by ClaudioFloreani** [reply]
Thank you Mike.
